cmd/respawn/respawn.go

```diff
@@ -69,6 +69,12 @@ func run(c *cli.Context) error {
 		panic(err)
 	}
 
+	// Now that we're booted, stop writing debug messages to the console
```

Comment:
would you? I wasn't sure :)

if its in console_init, does that mean it'll happen any time a `ros console switch` happens?

I went for respawn because i figured that runs before console, so if the user wants to over-ride it, they can use a `runcmd` or run it manually, and then it'll stay until reboot (or respawn dies)would you? I wasn't sure :)

if its in console_init, does that mean it'll happen any time a `ros console switch` happens?

I went for respawn because i figured that runs before console, so if the user wants to over-ride it, they can use a `runcmd` or run it manually, and then it'll stay until reboot (or respawn dies)